packages/components/stories/toast/toast.stories.js

```diff
@@ -29,7 +29,9 @@ stories
                     onClick={action('clicked')}
                     is_open={boolean('is open?', true)}
                     timeout={number('Timeout', 3000)}
-                    onClose={console.log('Closed')}
+                    onClose={() => {
+                        // console.log('Closed')
```

Comment:
i think you can keep it empty
or even you can write a comment instead of this commented out console log
something like ` // onClose callback`